specification/storagesync/resource-manager/readme.md

```diff
@@ -26,18 +26,27 @@ These are the global settings for the Storage Sync API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-03-01
+tag: package-2019-06
```

Comment:
shouldn't this be 2019-06-01?

Reply:
Thanks for pointing it out @ryansbenson 
I'll fix it.

Reply:
@Juliehzl please approve the following PR:
https://github.com/Azure/azure-rest-api-specs/pull/7413

Looks like I don't have permission to push to this branch, so I had to create a PR to fix the issue in the readme file.

Reply:
This is not fixed yet.

Reply:
I don't have permissions to push to this branch, so the fixes are going to this PR:
https://github.com/Azure/azure-rest-api-specs/pull/7413